ReactAndroid/src/main/jni/react/JSCHelpers.cpp

```diff
@@ -56,12 +56,14 @@ JSValueRef evaluateScript(JSContextRef context, JSStringRef script, JSStringRef
     auto line = exception.asObject().getProperty("line");
 
     std::ostringstream locationInfo;
-    std::string file = String::adopt(source).str();
-    locationInfo << "(" << (file.length() ? file : "<unknown file>");
-    if (line != nullptr && line.isNumber()) {
-      locationInfo << ":" << line.asInteger();
+    if (source != nullptr) {
+      std::string file = String::adopt(source).str();
+      locationInfo << "(" << (file.length() ? file : "<unknown file>");
```

Comment:
We should probably take the code path that prints `<unknown file>` -- this might make it easier for log analyzers to consume.